package.json

```diff
@@ -22,15 +22,16 @@
   },
   "homepage": "https://github.com/atlassian-labs/react-loosely-lazy#readme",
   "scripts": {
-    "types": "tsc --noEmit && flow --max-warnings=0",
-    "types:watch": "npm run type-check -- --watch",
-    "build": "npm run build:clean && npm run build:types-cjs && npm run build:types-esm && npm run build:cjs && npm run build:esm && npm run build:server",
+    "types:check": "tsc --noEmit && flow --max-warnings=0",
```

Comment:
Can we run `sort-package-json` over this file?

Reply:
Didn't know about this, quite cool lol